kafka-rest/src/test/java/io/confluent/kafkarest/integration/ClusterTestHarness.java

```diff
@@ -143,6 +143,8 @@ public static int choosePort() {
   protected Server restServer = null;
   protected String restConnect = null;
 
+  private static final long SLEEP_MS = 1000;
```

Comment:
nit: Let's add an `L` suffix to make this an actual `long` primitive. I personally also think this should be called `ONE_SECOND_IN_MILLIS` or `ONE_SECOND_MS`.

Reply:
I left it without the time value in the name because I wonder if we will need to increase the sleep time in the future, it's an arbitrary length sleep, and it may not be right at 1s.

But, the flip side is, if we have to do that we can always rename it to SLEEP_MS so I don't have strong feeling.

Reply:
Yup, I did change it already to say ONE_SECOND_MS

![image](https://user-images.githubusercontent.com/87994408/154318201-ff68322a-16fd-49fc-a88b-447fd6e8c52f.png)

I wonder if git is having a moment....